src/components/InputFile/InputFile.tsx

```diff
@@ -148,105 +131,54 @@ const FileList = styled.ul<{ themes: Theme }>(({ themes }) => {
   `
 })
 
-const FileButtonWrapper = styled.div<{ themes: Theme }>(({ themes }) => {
-  const { color, interaction } = themes
-
+const InputLabel = styled.label<{ themes: Theme }>(({ themes }) => {
```

Comment:
どういったファイルをアップロードするかのラベルでこのコンポーネントをラップすると考えると `label` のままだと

`label > label > input` の構造となってしまうため、ここは `<span>` でお願いします。手元で確認したところ、ポインティングデバイス、キーボードでも問題なく動作していました。`props.label` の値が `input[type="file"]` の `name` になりませんが、それは外から label などで参照すべきものなのでとりあえず良いかなと思います。

```suggestion
const InputLabel = styled.span<{ themes: Theme }>(({ themes }) => {
```